data-visualization/json/index04.js

```diff
@@ -0,0 +1,9 @@
+let myTeam = {
+  "Team": "Argentina",
+  "Sport": "Football",
+  "Year": 1986
+};
+
+/*
+Add another key-value pair after `Year` for `World Cup Winner`, this time use a Boolean value set to `true`.
```

Comment:
Add a line here about how properties can have spaces within them. This is unique to JSON (as compared to regular JS objects) so it would be good to cover that here.

Additionally:
```suggestion
Add another key-value pair after `Year` for `World Cup Winner`, this time use a Boolean value set to `true` (remember not to surround this value with quotation marks).
```